ophyd/runengine/runengine.py

```diff
@@ -191,28 +257,46 @@ def _get_data_keys(self, **kwargs):
                 names += [o.name for o in det.signals]
             else:
                 names.append(det.name)
-
         return names
 
     def start_run(self, runid, begin_args=None, end_args=None, scan_args=None):
-        # create run_header and event_descriptors
-        header = data_collection.create_run_header(scan_id=runid, **scan_args)
-        # header = {'run_header': 'Foo'}
+        """
+
+        Parameters
+        ----------
+        runid : sortable
+        begin_args
+        end_args
+        scan_args
+
+        Returns
+        -------
+        data : dict
+            {data_name: []}
+        """
+        # format the begin run event information
+        beamline_id = scan_args.get('beamline_id', None)
```

Comment:
This will raise an exception in the cause of the default `scan_args` value.  I suggest either making it mandatory (my relatively uninformed preference) or adding the 

```python
if scan_args is None:
    scan_args = dict()
```